image_tools/src/cam2image.cpp

```diff
@@ -58,7 +58,11 @@ int main(int argc, char * argv[])
 
   auto node = rclcpp::node::Node::make_shared("cam2image");
 
-  auto pub = node->create_publisher<sensor_interfaces::msg::Image>("image", 10);
+  rmw_qos_policies_t custom_qos_policies = rmw_qos_policies_default;
+  custom_qos_policies.depth = 10;
```

Comment:
It should be, but I didn't want to make more changes than necessary, just update the code for the QoS changes.